src/pageObjects/WelcomePage.js

```diff
@@ -0,0 +1,44 @@
+
+
+
+export default class WelcomePage {
```

Comment:
I believe you are busy but please create BasePage and use inheritance power. If you need some tips you can review code from our lesson or ping me